### ISSUE TITLE
can you show a video demo?

### ISSUE THREAD

**Issue by luoww1992** [muelea/buddi#3]
it can be real time if not vis it, how many fps ?

**Comment by muelea** [comment]
No, this method does not run in real time. Diffusion models are known to be slow at inference. To fit meshes to images we use optimization which does also not run in realtime.

**Comment by LinpeiSTU** [comment]
Thanks for your great work! I am also curious about the video demo but not for real time. Is it robust for videos and will the motion be smooth?
